src/status_im2/common/scroll_page/view.cljs

```diff
@@ -134,7 +134,7 @@
       [:<>
        [:f> scroll-page-header @scroll-height height name page-nav-right-section-buttons
         cover-image sticky-header top-nav title-colum navigate-back?]
-       [rn/scroll-view
+       [rn/flat-list
```

Comment:
hmm, why so? 🤔

Reply:
cool, thanks @jo-mut

Reply:
@J-Son89 usage of scroll-view should generally be avoided in RN. Always use FlatList or SectionList. Here is a quick comparison: https://stackoverflow.com/questions/55256221/flatlist-vs-scrollview

Reply:
thanks @OmarBasem :D